src/main/java/com/stories/model/TaskModel.java

```diff
@@ -0,0 +1,18 @@
+package com.stories.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.Data;
+
+@Data
+@Document(collection = "stories")
+public class TaskModel {
+
+	private String _id;
```

Comment:
Is not necessary add a @ID for mongoDB ?

Reply:
fixed